#165364245 Fix broken signup and signin links

tohbay opened the pull request:
### What does this PR do?
This PR fixes the broken signup and signin links

### Description of Task to be completed?
#### Have the following working:
- ensure that when a user clicks on the signup button after filling the signup form, the user is greeted with the user profile page.
- ensure that when a user clicks on the signin button after filling the signin form, the user is greeted with the user profile page.

### How should this be manually tested?
#### The following steps can be used:
- Clone the repository or navigate to the App URL on github pages.
- Signup or Signin with your user credentials.
- The application will take you to the client.html page

### Any background context you want to provide?
- User should ensure there is internet on their machine to access the pages.

### What are the relevant pivotal tracker stories?
[Bugfix #165364245]: https://www.pivotaltracker.com/story/show/165364245

tohbay committed:
fix(signupPage): fix broken signup link

- ensure a user is linkned to user-profile page after clicking on signup button,

[Fixes #165364245]

AdinoyiSadiq commented:
Hi Tobechukwu, 

- Please ensure that you are following the proper format for the git commit messages as well as the PR title.
- Also please ensure that the you PR has complete sentences in them. Instead of "Fix broken signup and signin links", it should be "This PR fixes the broken signup and signin links"

tohbay committed:
fix(signinPage): fix broken signin link

- ensure a user is served the user profile page after clicking on signin button,

[Fixes #165364245]
